pkg/visor/init_cli.go

```diff
@@ -0,0 +1 @@
+package visor
```

Comment:
Why do we require this file and the `init_cli.go` pendant?

Reply:
good catch, It was an accidental committed file, I'll remove it.